egs/librispeech/asr/simple_v1/mmi_mbr_decode.py

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/env python3
+
+# Copyright (c)  2020  Xiaomi Corporation (authors: Daniel Povey, Haowen Qiu)
+# Apache 2.0
+
```

Comment:
At some point we may want to investigate a more generic decoding script, but this is OK for now..